datasets/mac_morpho/README.md

```diff
@@ -0,0 +1,172 @@
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [Mac-Morpho homepage](http://nilc.icmc.usp.br/macmorpho/)
+- **Repository:** [Mac-Morpho repository](http://nilc.icmc.usp.br/macmorpho/)
+- **Paper:** [Evaluating word embeddings and a revised corpus for part-of-speech tagging in Portuguese](https://journal-bcs.springeropen.com/articles/10.1186/s13173-014-0020-x)
+- **Point of Contact:** [Erick R Fonseca](mailto:erickrfonseca@gmail.com)
+
+### Dataset Summary
+
+Mac-Morpho is a corpus of Brazilian Portuguese texts annotated with part-of-speech tags. 
+Its first version was released in 2003 [1], and since then, two revisions have been made in order 
+to improve the quality of the resource [2, 3].
+The corpus is available for download split into train, development and test sections. 
+These are 76%, 4% and 20% of the corpus total, respectively (the reason for the unusual numbers 
+is that the corpus was first split into 80%/20% train/test, and then 5% of the train section was 
+set aside for development). This split was used in [3], and new POS tagging research with Mac-Morpho 
+is encouraged to follow it in order to make consistent comparisons possible.
+
+
+[1] Aluísio, S., Pelizzoni, J., Marchi, A.R., de Oliveira, L., Manenti, R., Marquiafável, V. 2003. 
+An account of the challenge of tagging a reference corpus for brazilian portuguese. 
+In: Proceedings of the 6th International Conference on Computational Processing of the Portuguese Language. PROPOR 2003
+
+[2] Fonseca, E.R., Rosa, J.L.G. 2013. Mac-morpho revisited: Towards robust part-of-speech. 
+In: Proceedings of the 9th Brazilian Symposium in Information and Human Language Technology – STIL
+
+[3] Fonseca, E.R., Aluísio, Sandra Maria, Rosa, J.L.G. 2015. 
+Evaluating word embeddings and a revised corpus for part-of-speech tagging in Portuguese. 
+Journal of the Brazilian Computer Society.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+Portuguese
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```

Comment:
can you provide one example from the dataset here please ?

You can get one by using `load_dataset` and taking the first sample `dataset["train"][0]` for example

Reply:
Done!

Reply:
Hi @lhoestq, I have a doubt. Why do I need to provide an instance example of this dataset but not for the others I've added? Is there some particularity about this kind of dataset?

Reply:
We should have an example in as many dataset cards as possible. We've recently noticed that it was missing in too many dataset cards so we're now asking them explicitly. Feel free to add them on the other datasets you've added though.